env/src/test/java/com/sap/cloud/security/config/EnvironmentsCFTest.java

```diff
@@ -16,32 +16,30 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class EnvironmentsTest {
+class EnvironmentsCFTest {
 
 	private final InputStream vcapMultipleXsuaa;
 
-	public EnvironmentsTest() throws IOException {
+	EnvironmentsCFTest() throws IOException {
 		vcapMultipleXsuaa = IOUtils.toInputStream(
 				IOUtils.resourceToString("/vcapXsuaaServiceMultipleBindings.json", UTF_8), UTF_8);
 	}
 
 	@Test
-	public void getCurrent_returnsOnlySingleInstance() {
+	void getCurrent_returnsOnlySingleCFInstance() {
 		Environment firstEnvironment = Environments.getCurrent();
 		Environment secondEnvironment = Environments.getCurrent();
 
 		assertThat(firstEnvironment).isSameAs(secondEnvironment);
 	}
 
 	@Test
-	public void getCurrent_returnsCorrectEnvironment() {
-		// TODO 29.11.19 c5295400: extend test when more than one environment is
-		// supported
+	void getCurrent_returnsCf() {
```

Comment:
Cf or CF abbreviation should be consistent throughout all tests and for this test I think it suit even better to rename it to getType_returnsCF